src/types/Goal.ts

```diff
@@ -0,0 +1,4 @@
+export default interface Goal {
+  id: string;
+  name: string;
+}
```

Comment:
This is just a bare bones schema for the goal object right now. I imagine we would want to add to this things like `anticipatedTime`, `currentTime`, etc. to this later

Reply:
That's what I was thinking too we can implement that later down the line when we have the timer functionality working.